notebooks/pixel_classification_api/environment.yml

```diff
@@ -4,5 +4,6 @@ channels:
   - conda-forge
 dependencies:
   - ilastik-dependencies-no-solvers
+  - ilastik-meta >=1.4.0b14
   - notebook
```

Comment:
Does it make sense to recommend JupyterLab instead of just notebooks?

Reply:
I don't see added value in jupyterlab for the purpose of this notebook and would in general try to keep this minimal.